lib/cartopy/tests/crs/test_transverse_mercator.py

```diff
@@ -21,16 +22,56 @@ def setup_class(self):
         self.point_b = (0.5, 50.5)
         self.src_crs = ccrs.PlateCarree()
 
+    def check_args(self, approx, proj, other_args):
+        if ccrs.PROJ4_VERSION < (6, 0, 0):
```

Comment:
Can get rid of this check now since we are dependent on pyproj's PROJ version and looks like we are at 8+
https://pyproj4.github.io/pyproj/3.3.1/